pkg/kfapp/ossm/types.go

```diff
@@ -28,23 +24,52 @@ type manifest struct {
 	processed bool
 }
 
-func (m *manifest) targetPath() string {
-	return fmt.Sprintf("%s%s", m.path[:len(m.path)-len(filepath.Ext(m.path))], ".yaml")
+// In order to process the templates, we need to create a tmp directory
+// to store the files. This is because embedded files are read only.
+var outputDir = "/tmp/ossm-installer/"
```

Comment:
```suggestion
const outputDir = "/tmp/ossm-installer/"
```

Reply:
Maybe rename it also to `baseOutputDir` (don't want to make it a suggestion, as that would lead to compilation failures)